tests/api-test/getSecret.bru

```diff
@@ -5,11 +5,25 @@ meta {
 }
 
 get {
-  url: http://127.0.0.1:8000/secret/482820c2-9170-41c6-b5c3-5f20ebeb58c6?meta=true
+  url: http://127.0.0.1:8000/secret/{{id}}?meta=true
```

Comment:
Nitpick: can we have  ip from an enviroment variable, so in future we can direct to prod  in the release workflow?

Reply:
Sure. I should have done that from the start. Done.

Reply:
I also added another way of testing. You can take a look at "getSecret" request in "Assert" tab.